source/actions.js

```diff
@@ -1,14 +1,17 @@
 
 import {
+    START,
     SET,
     SET_PROFILE,
     LOGIN,
     LOGOUT,
     LOGIN_ERROR,
-    NO_VALUE
+    NO_VALUE,
+    INIT_BY_PATH
 } from './constants'
 
 import { Promise } from 'es6-promise'
+var moment = require('moment');
```

Comment:
I would like to avoid the dependency on moment if possible. Is there a reason we can't just use javascript's `Date.now()`? It is just for redux right? So it doesn't matter too much the format of the time?